Packs/Legacy/Scripts/SetGridField/SetGridField.py

```diff
@@ -0,0 +1,200 @@
+# STD Libaries
+from typing import Optional, List, Dict, Any
+# 3-rd party libaries
+import pandas as pd
+import phrases_case
+# Local packages
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+
+def normalized_string(phrase: str) -> str:
+    """ Normalize columns or Grid to connected word in lower-case.
+
+    Args:
+        phrase: Phrase to normalize.
+
+    Returns:
+        str: Normalized phrase.
+
+    Examples:
+        >>> normalized_string("TestWord")
+        "testword"
+        >>> normalized_string("hello_world")
+        "hello_world"
+    """
+    return phrases_case.camel(phrase).replace("'", "").lower()
+
+
+def filter_the_dict(dict_obj: Dict[Any, Any], keys: List[str], max_keys: Optional[int] = None) -> Dict[Any, Any]:
+    """ Filter keys from Dictionary:
+            1. Will save all keys in keys parameter.
```

Comment:
`keys` does not change in the code (and it shouldn't be)

Reply:
I wrote more clear description